examples/echo/dialer.js

```diff
@@ -34,7 +34,14 @@ async.parallel([
 
   const peerListener = new PeerInfo(ids[1])
   idListener = ids[1]
-  peerListener.multiaddr.add(multiaddr('/ip4/127.0.0.1/tcp/10333'))
+
+  let toDial = '/ip4/127.0.0.1/tcp/10333'
```

Comment:
It's a leftover from how it was before. Example seems to work anyways. You want me to add it though, since the ID in our echo example is always the same anyways?